src/c4/ofpstream.cc

```diff
@@ -0,0 +1,70 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   c4/ofpstream.cc
+ * \author Mike Buksas
+ * \date   Thu May  1 14:42:10 2008
+ * \brief
+ * \note   Copyright (C) 2016-2018 Los Alamos National Security, LLC.
+ *         All rights reserved.
```

Comment:
* Please update all of this header stuff (author, creation date, etc.)
* Ditto for the .hh file.

Reply:
Done.